packages/jira-adapter/src/filters/workflow_scheme.ts

```diff
@@ -140,12 +140,51 @@ export const preDeployWorkflowScheme = async (
   }
 }
 
+const replaceIDsWithNames = (messages: string[], ids: string[],
+  idToInstance: Record<string, InstanceElement>):string[] =>
```

Comment:
styling nit:
```suggestion
const replaceIDsWithNames = (
  messages: string[],
  ids: string[],
  idToInstance: Record<string, InstanceElement>,
):string[] =>
```
also - since there are two `strings[]` params, consider using named params?